unit_test/test_tag_trimming.py

```diff
@@ -60,14 +60,21 @@ def test_non_matching(self):
 
         out = init
 
-        self.assertEqual(tag_trimming.trim_tags(init, "off"), out)
+        self.assertEqual(tag_trimming.trim_html_tags(init, "off"), out)
 
     def test_curly_tags(self):
         init = '<0>foo</0>{:target="_blank"}'
 
         out = init
 
-        self.assertEqual(tag_trimming.trim_tags(init, "off"), out)
+        self.assertEqual(tag_trimming.trim_html_tags(init, "off"), out)
+
+    def test_inside_backquotes(self):
```

Comment:
It turns out that for markdown code it was never a problem because our code matches only the most outer formatting tags, then their content is trimmed and no further recursive analysis is performed.